public/views/app.devices.html

```diff
@@ -66,4 +66,176 @@ <h5>{{ device.name }}</h5>
+                    <div class="col-sm-7 fields-space">
+                      <input type="text" class="form-control" ng-model="form.dataItem.schema.sensorName" placeholder="Sensor Name" required>
+                    </div>
+                    <label class="control-label col-sm-4" for="typeID">Type ID*:</label>
+                    <div class="col-sm-8 fields-space">
+                      <select class="form-control" ng-change="onTypeIDChange()" ng-model="form.dataItem.schema.typeID" required>
+                        <option value="0" selected>None</option>
+                        <option value="1" >Voltage</option>
+                        <option value="2" >Current</option>
+                        <option value="3" >Resistance</option>
+                        <option value="4">Power</option>
+                        <option value="5">Temperature</option>
+                        <option value="6">Relative Humidity</option>
+                        <option value="7">Luminosity</option>
+                        <option value="8">Time</option>
+                        <option value="9">Mass</option>
+                        <option value="10">Pressure</option>
+                        <option value="11">Distance</option>
+                        <option value="12">Angle</option>
+                        <option value="13">Volume</option>
+                        <option value="14">Area</option>
+                        <option value="15">Rain</option>
+                        <option value="16">Density</option>
+                        <option value="17">Latitude</option>
+                        <option value="18">Longitude</option>
+                        <option value="19">Speed</option>
+                        <option value="20">Volumeflow</option>
+                        <option value="21">Energy</option>
+                        <option value="65520">Presence</option>
+                        <option value="65521">Switch</option>
+                        <option value="65522">Command</option>
+                        <option value="65296">Analog</option>
+                      </select>
+                    </div>
+                    <label class="control-label col-sm-4" for="unit">Unit*:</label>
+                    <div class="col-sm-8 fields-space">
+                      <select class="form-control" ng-model="form.dataItem.schema.unit" required>
+                        <option ng-repeat="option in unitValues[form.dataItem.schema.typeID]"
+                        value="{{ option.value }}">{{ option.name }}</option>
+                      </select>
+                    </div>
+                    <label class="control-label col-sm-4" for="valueType">Value Type*:</label>
+                    <div class="col-sm-8 fields-space">
+                      <select class="form-control" ng-model="form.dataItem.schema.valueType" required>
+                        <option value="1" selected>INT</option>
```

Comment:
Couldn't they be lowercase as you did with type id's names?

Reply:
Done!